GPU/Common/TextureScalerCommon.cpp

```diff
@@ -698,22 +698,22 @@ const int MIN_LINES_PER_THREAD = 4;
 
 void TextureScalerCommon::ScaleXBRZ(int factor, u32* source, u32* dest, int width, int height) {
 	xbrz::ScalerCfg cfg;
-	ParallelRangeLoop(&g_threadManager, std::bind(&xbrz::scale, factor, source, dest, width, height, xbrz::ColorFormat::ARGB, cfg, std::placeholders::_1, std::placeholders::_2), 0, height, MIN_LINES_PER_THREAD);
+	ParallelRangeLoop(&g_threadManager, [factor, source, dest, width, height, cfg](auto && PH1, auto && PH2) { return xbrz::scale(factor, source, dest, width, height, xbrz::ColorFormat::ARGB, cfg, std::forward<decltype(PH1)>(PH1), std::forward<decltype(PH2)>(PH2)); }, 0, height, MIN_LINES_PER_THREAD);
```

Comment:
This doesn't look very good. Does this have any huge advantage over just using std::bind ?

Reply:
> This doesn't look very good. Does this have any huge advantage over just using std::bind ?

All answers in https://stackoverflow.com/questions/24852764/stdbind-vs-lambda-performance